Explorer/Assets/DCL/ResourcesUnloading/UnloadStrategies/AggressiveUnloadStrategy.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using Cysharp.Threading.Tasks;
+using ECS.Prioritization;
+using UnityEngine;
+
+namespace DCL.ResourcesUnloading.UnloadStrategies
+{
+    public class AggressiveUnloadStrategy : IUnloadStrategy
+    {
+        public bool isRunning { get; set; }
+
+        private readonly IRealmPartitionSettings realmPartitionSettings;
+        private const int FORCE_UNLOAD_DURING_THIS_FRAMES = 200;
+
+        public AggressiveUnloadStrategy(IRealmPartitionSettings realmPartitionSettings)
+        {
+            this.realmPartitionSettings = realmPartitionSettings;
+        }
+
+        public void TryUnload(ICacheCleaner cacheCleaner)
+        {
+            if (isRunning)
+                return;
+            
+            isRunning = true;
+            //Forces MaxLoadingDistanceInParcels to the minimum value
+            //TODO (Juani): A message warning that the distance has been silently modified
+            realmPartitionSettings.MaxLoadingDistanceInParcels = realmPartitionSettings.MinLoadingDistanceInParcels;
```

Comment:
The intention is actually to load already loaded scenes and LODs; not to prevent the loading of new assets.
By taking the LoadingDistance to the minimum, all scenes that were loaded above the minimum should start to unload